app/components/utilities/AAAListButton.tsx

```
@@ -0,0 +1,121 @@
+        open={isOpen}
+        onClose={() => setIsOpen(false)}
+        className="fixed inset-0 flex items-center justify-center p-4">
+        <Dialog.Overlay className="fixed inset-0 bg-black/50" />
+        <div className="relative bg-white rounded-lg p-6 max-w-sm w-full">
+          <h3 className="text-lg font-bold mb-4">AAA List Settings</h3>
+
+          <div className="space-y-4">
+            <div>
+              <label className="block text-sm font-medium text-gray-700">
+                Minimum Market Value
+              </label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)